tests/test_modules.py

```diff
@@ -105,7 +105,7 @@ def test_modules_remove_fastqc(self):
     def test_modules_remove_fastqc_alternative_source(self):
         """Test removing FastQC module after installing it from an alternative source"""
         self.mods_alt.install("fastqc")
-        module_path = os.path.join(self.mods.pipeline_dir, "modules", "ewels", "software", "fastqc")
+        module_path = os.path.join(self.mods.pipeline_dir, "modules", "external", "software", "fastqc")
```

Comment:
```suggestion
        module_path = os.path.join(self.mods.pipeline_dir, "modules", "external", "fastqc")
```

Reply:
I guess not, keeps things a bit cleaner.